plugins/outputs/stomp/stomp.go

```diff
@@ -0,0 +1,95 @@
+package stomp
+
+import (
+	ts "crypto/tls"
+	"net"
+
+	"github.com/go-stomp/stomp"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+//STOMP ...
+type STOMP struct {
+	Host      string `toml:"host"`
+	Username  string `toml:"username,omitempty"`
+	Password  string `toml:"password,omitempty"`
+	QueueName string `toml:"queueName"`
+	SSL       bool   `toml:"ssl"`
+	tls.ClientConfig
+	Conn      *ts.Conn
+	NetConn   net.Conn
+	Stomp     *stomp.Conn
+	serialize serializers.Serializer
+}
+
+//Connect ...
+func (q *STOMP) Connect() error {
+	var err error
+	if q.SSL == true {
+		tlsConfig, _ := q.ClientConfig.TLSConfig()
+
+		q.Conn, err = ts.Dial("tcp", q.Host, tlsConfig)
+
+		q.Stomp, err = stomp.Connect(q.Conn, stomp.ConnOpt.HeartBeat(0, 0), stomp.ConnOpt.Login(q.Username, q.Password))
+	} else {
+		q.NetConn, err = net.Dial("tcp", q.Host)
+		q.Stomp, err = stomp.Connect(q.NetConn, stomp.ConnOpt.HeartBeat(0, 0), stomp.ConnOpt.Login(q.Username, q.Password))
+
+	}
+	if err != nil {
+		println("cannot connect to server", err.Error())
```

Comment:
add `	Log telegraf.Logger `toml:"-"`` to the STOMP struct instead, and Telegraf will give you a logger instance to use. in tests, set it to `testutil.Logger{}`